config/logging.php

```diff
@@ -48,36 +48,38 @@
 	'channels' => [
 		'stack' => [
 			'driver' => 'stack',
-			'channels' => ['debug-daily', 'notice', 'warning', 'error', ],
+			'channels' => ['debug-daily', 'error', 'warning',  'notice'],
```

Comment:
I haven't actually checked how this part works, but wouldn't we want debug at the notice end?

Reply:
Not really.
- Debug logs everything
- then you hit Error, if the message is an error it stops here
- then you hit Warning, same principle
- then you hit notice.

If you go the other way, the problem is that Warning will also capture Error messages. which makes it very noisy.
And you can't block propagation on Warning because Error will also be stopped.

Reply:
Ah, I see. :+1: